docker-compose.yml.staging

```diff
@@ -37,6 +37,7 @@ services:
     volumes:
       - postgres_data:/var/lib/postgresql/data/
     restart: always
+    command: postgres -c 'config_file=/etc/postgresql/postgresql.conf'
```

Comment:
Please add this line to `docker-compose.yml.development` too!

Reply:
do we want that? Weren't these determined for the servers only?

Reply:
I think we want the configuration to be as similar as possible between develop, staging and production, don't we?

Reply:
I can see where you're coming from, though. 